schedule/containers/sle_image_on_sle_host.yaml

```diff
@@ -12,6 +12,17 @@ schedule:
   - containers/container_diff
   - '{{validate_btrfs}}'
 conditional_schedule:
```

Comment:
would you move this section above schedule as in the other yamls?

Reply:
7f7d1aa26 done